.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -24,6 +24,7 @@ By opening this PR, I confirm that I have:
 - [ ] Accurately updated the [CHANGELOG](https://github.com/SMI/SmiServices/blob/master/CHANGELOG.md)
```

Comment:
Shouldn't these be relative paths?

Reply:
Potentially... I think relative paths probably expand to the diff between the source and target branch when used in PRs? This is just a link to the file though, and the summary should include a link to the diff anyway.

Reply:
Ok that makes sense.  It's a dead link atm which was confusing but once it's merged it will make more sense.